app/controllers/candidate_interface/referees_controller.rb

```diff
@@ -109,6 +114,20 @@ def review
       @application_form = current_candidate.current_application
     end
 
+    def complete
+      if current_application.application_references.count >= ApplicationForm::MINIMUM_COMPLETE_REFERENCES
+        current_application.update!(application_form_params)
+
+        redirect_to candidate_interface_application_form_path
+      else
+        flash[:warning] = 'You can’t mark this section complete without adding two referees.'
+        current_application.references_completed = false
+        @application_form = current_candidate.current_application
+
+        render :review
+      end
```

Comment:
Interesting. I wondered here for a bit if we should instead hide the checkbox until two references are added, but that could be confusing to users.